### ISSUE TITLE
Failed to retrieve Access Token

### ISSUE THREAD

**Issue by ttornqvist** [speleolontra/daikin_residential_altherma#124]
Hi,

I have Altherma 3 with BRP069A78. It has been working with Onecta app for few months now. I cannot setup connection because of error below. It doesn't matter if I use yaml or UI configuration.

Logger: custom_components.daikin_residential_altherma.config_flow
Source: custom_components/daikin_residential_altherma/config_flow.py:73
Integration: Daikin Residential Controller for Altherma ([documentation](https://github.com/speleolontra/daikin_residential_altherma), [issues](https://github.com/speleolontra/daikin_residential_altherma/issues))
First occurred: 16:44:18 (1 occurrences)
Last logged: 16:44:18
Failed to retrieve Access Token: ('Error trying to extract API version: %s', TypeError('get() takes 1 positional argument but 2 were given'))

I have googled and read about this, and I can confirm following:
- time is correct in both HA and Altherma (although I believe time errors are between Daikin cloud and HA, not actual pump).
- I have rebooted HA, and also Proxmox I use to host it

HA version: Home Assistant 2023.3.4 Supervisor 2023.03.1 Operating System 9.5 Frontend 20230309.0
And few minutes ago I updated this integration to 2.5.1. Still same error.

**Comment by jwillemsen** [comment]
I have not seen this one before, are you maybe using an older requests package, see https://github.com/h4de5/home-assistant-toshiba_ac/issues/87 and https://github.com/rospogrigio/daikin_residential/issues/84

**Comment by ttornqvist** [comment]
Thanks!! It was that. I have also Toshiba AC and I installed it first from HACS, and it worked. Then I started to fight over Daikin. I removed Toshiba AC (devices, and integration) and lo and behold, there it is now. Now Daikin works.
It would be great if these can coexists, but I do have Tado for my Toshiba, and it has started to work with HA also (and with HomeKit it has worked all the time), so now I keep Daikin. Thank you once more, I didn't find those issues by myself.
